src/providers/reka-ai/index.ts

```diff
@@ -0,0 +1,16 @@
+import { ProviderConfigs } from '../types';
+import RekaAIApiConfig from './api';
+import {
+  RekaAIChatCompleteConfig,
+  RekaAIChatCompleteResponseTransform,
+} from './chatComplete';
+
+const RekaAIConfig: ProviderConfigs = {
+  chatComplete: RekaAIChatCompleteConfig,
```

Comment:
They don't seem to have a streaming endpoint so I only added chat completion support.

Reply:
Acknowledged. We have a json to stream function but we can use that in another PR.